lib/init.g

```diff
@@ -421,7 +421,7 @@ end );
 
 BindGlobal( "ShowKernelInformation", function()
   local sysdate, linelen, indent, btop, vert, bbot, print_info,
-        config, str, gap;
+        config, str, gap, tmp;
```

Comment:
was this left in from some temporary work?

Reply:
Something like that (it'll be in a different PR)